kimchi/src/prover.rs

```diff
@@ -398,7 +395,7 @@ where
         let alpha_chal = ScalarChallenge(fq_sponge.challenge());
```

Comment:
it'd be nice to give the same treatment to the Fq-Sponge, Fr-Sponge, fq_sponge, fr_sponge at some point :)